### ISSUE TITLE
Click-to-ping someone with an emoji in their name doesn't create a pill

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#4581]
### Description

![image](https://user-images.githubusercontent.com/1190097/28218926-5b2c1b8a-6877-11e7-8aab-e6790fc9b2bd.png)

Blurred for privacy where possible. Red emphasis is to highlight the issue.

### Steps to reproduce

- Have an emoji in your name
- Click your name to ping yourself

Log: not sent    

### Version information

- **Platform**: web (in-browser)
- **Browser**: Chrome 58
- **OS**: Windows 10
- **URL**: riot.im/develop

**Comment by lampholder** [comment]
There's something about the ☎️ emoji that is being handled differently to the 🕹 emoji (that I randomly had as part of my display name before).

The RTE is certainly capable of displaying ☎️, as evidenced if you type :telephone:, but it's not rendering in the editor when pasted in. It _does_ appear in the timeline though.

To reproduce, copy the below, paste it into the new RTE, note the absense of the telephone in the editor, press send, note the presence of the telephone in the timeline.
```
Why can I see the joystick (🕹) but not the classy old-skool telephone (☎️)?
```

**Comment by lukebarnard1** [comment]
This is _probably_ because emojis have entities attached to them to make them immutable.

**Comment by turt2live** [comment]
They now show up, however they are not pilled

**Comment by ara4n** [comment]
i think this has been fixed in matrix-org/matrix-react-sdk#1890 if someone can check (i can't find a good one to test against)

**Comment by t3chguy** [comment]
Yup!
![image](https://user-images.githubusercontent.com/2403652/42778694-e01d8a1a-8935-11e8-865f-42f24fdc0636.png)
